From d6a9eb1f5db3a3204206c6806492d4c14eab1ff5 Mon Sep 17 00:00:00 2001
From: Adam YH Lee <adam@gumstix.com>
Date: Tue, 4 Mar 2014 12:01:13 -0800
Subject: [PATCH] [virtual/kernel] Disable uAP initialization

Network Manager does not know how to manage multiple interfaces
of SD8787. uAP mode has been therefore disabled. 

---
 drivers/net/wireless/mwifiex/main.c |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/net/wireless/mwifiex/main.c b/drivers/net/wireless/mwifiex/main.c
index 4680362..7e8031b 100644
--- a/drivers/net/wireless/mwifiex/main.c
+++ b/drivers/net/wireless/mwifiex/main.c
@@ -364,12 +364,6 @@ static void mwifiex_fw_dpc(const struct firmware *firmware, void *context)
 		goto err_add_intf;
 	}
 
-	/* Create AP interface by default */
-	if (!mwifiex_add_virtual_intf(adapter->wiphy, "uap%d",
-				      NL80211_IFTYPE_AP, NULL, NULL)) {
-		dev_err(adapter->dev, "cannot create default AP interface\n");
-		goto err_add_intf;
-	}
 	rtnl_unlock();
 
 	mwifiex_drv_get_driver_version(adapter, fmt, sizeof(fmt) - 1);
-- 
1.7.9.5

